plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -118,7 +118,7 @@
 		  scope="total"
 		  class="com.google.cloud.tools.eclipse.appengine.validation.AppEngineWebXmlSourceValidator"
 		  id="appEngineWebXmlSourceValidator">
-		<contentTypeIdentifier id="appengine-web.xml">
+		<contentTypeIdentifier id="com.google.appengine.dd.xml">
```

Comment:
It seemed like an App Engine deployment descriptor to me!  I'm open to better names.